TaylorIntegrationTests/TaylorIntegrationTests.swift

```diff
@@ -17,7 +17,7 @@ enum IntegrationTestsError: Error {
 
 class TaylorIntegrationTests: QuickSpec {
     
-    var runTaskPath : String {
+    var taylorExecutablePath : String {
```

Comment:
How about `executableFilePath`?
If we rename the tool, we don't want to be needed to make too many changes.